x-pack/docs/en/rest-api/security/disable-users.asciidoc

```diff
@@ -0,0 +1,44 @@
+[role="xpack"]
+[[security-api-disable-user]]
+=== Disable users API
+
+Disables users in the native realm. 
+
+
+==== Request
+
+`PUT /_xpack/security/user/<username>/_disable` 
+
+
+==== Description
+
+By default, when you create users, they are enabled. 
+//TBD: What is disallowed when a user is disabled? Is it just intended for 
```

Comment:
Yes it a way to revoke access for a user and prevent them from accessing elasticsearch